src/modules/trpc/procedure-with-graphql-client.ts

```diff
@@ -21,6 +22,12 @@ const attachAppToken = middleware(async ({ ctx, next }) => {
     });
   }
 
+  try {
+    await verifyJWT({ appId: authData.appId, token: ctx.token, apiUrl: saleorApiUrl });
```

Comment:
```suggestion
    await verifyJWT({ appId: authData.appId, token: ctx.token, apiUrl: saleorApiUrl });
```

Does this part works? Can't see `saleorApiUrl` defined in the scope

Reply:
it doesnt, I need to wait for new app-sdk to provide it